lib/bolt/module_installer/installer.rb

```diff
@@ -1,19 +1,20 @@
 # frozen_string_literal: true
 
-require 'r10k/cli'
 require 'bolt/r10k_log_proxy'
 require 'bolt/error'
 
 # This class is used to install modules from a Puppetfile to a module directory.
 #
 module Bolt
-  class Puppetfile
+  class ModuleInstaller
     class Installer
       def initialize(config = {})
         @config = config
       end
 
       def install(path, moduledir)
+        require 'r10k/cli'
```

Comment:
Do we use this? Why do we need to require it now?

Reply:
It's always been required (https://github.com/puppetlabs/bolt/blob/main/lib/bolt/puppetfile/installer.rb#L3), it just needs to be loaded lazily since it requires some larger gems that we don't always want to load for every Bolt command. Alternatively, we can lazily load `bolt/module_installer/installer` as part of `Bolt::ModuleInstaller.install_puppetfile`.